core/integration/features/spot/orders/0080-SPOT-024.feature

```diff
@@ -98,4 +98,29 @@ Feature: Spot market bug (from the incentive) replication
     Then "party3" should have general account balance of "1000" for asset "USD"
     Then "party4" should have general account balance of "1" for asset "BTC"
 
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference | error |
+      | party3 | BTC/USD   | buy  | 20     | 10    | 0                | TYPE_LIMIT | TIF_GTC | p3-b2     |       |
+
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference | error |
+      | party4 | BTC/USD   | sell | 1      | 66000 | 0                | TYPE_LIMIT | TIF_GTC | p4-s2     |       |
+
+    #0080-SPOT-026: amend order - order is amended such that would trade immediately and the party can't afford none/some of the trades
+    When the parties amend the following orders:
+      | party  | reference | price | size delta | tif     | error                                                              |
+      | party3 | p3-b2     | 66000 | -10        | TIF_GTC | party does not have sufficient balance to cover the trade and fees |
```

Comment:
can we verify the the original orders are unchanged? and the balances in the holding and general account are as expected